packages/scss/src/components/indexTable/mods.scss

```diff
@@ -0,0 +1,332 @@
+        }
+    }
+}
+
+@mixin stickyHeader {
+    .indexTable-head {
+        position: sticky;
+        top: var(--components-indexTable-row-spacing);
+        z-index: 2;
+    }
+
+    //hide box-shadow from underneath rows that slighly appear on edges of the sticky header when scrolling
+    .indexTable-head-row-cell {
+        position: relative;
+        &::before,
+        &::after {
+            content: '';
+            position: absolute;
+            inset: calc(var(--components-indexTable-row-spacing) * -1) calc(var(--components-indexTable-padding) * -1) 0;
+            z-index: -1;
+        }
+        &::before {
+            background-color: var(--commons-background-base);
+        }
+        &::after {
+            background-color: var(--components-indexTable-background-color);
+        }
+        &:first-child::after {
+            border-top-left-radius: var(--components-indexTable-border-radius);
+        }
+        &:last-child::after {
+            border-top-right-radius: var(--components-indexTable-border-radius);
+        }
+    }
+}
+
+@mixin responsiveCardList {
+    display: flex;
+    flex-direction: column;
+    gap: var(--components-indexTable-row-spacing);
+    padding: var(--components-indexTable-cell-padding);
+
+    .indexTable-head {
+        display: none;
+    }
+
+    .indexTable-body {
+        display: flex;
+        flex-direction: column;
+    }
+
+    .indexTable-body-row,
+    .indexTable-foot-row {
+        position: relative;
+        display: grid;
+        // Grid layout by default is a basic one column template.
+        // The user can override this and build a specific layout by using --components-indexTable-row-responsive-grid-template-columns, --components-indexTable-row-responsive-grid-template-rows and --components-indexTable-row-responsive-grid-template-areas
+        grid-template-columns: var(--components-indexTable-row-responsive-grid-template-columns, 1fr);
+        grid-template-rows: var(--components-indexTable-row-responsive-grid-template-rows, none);
+        grid-auto-rows: auto;
+        grid-template-areas: var(--components-indexTable-row-responsive-grid-template-areas, none);
+        align-items: center;
+        padding: var(--components-indexTable-row-spacing-responsive) 0;
+        margin-bottom: var(--components-indexTable-row-spacing-responsive);
+        // "Title" lines that do not contain "card" cells are smaller
+        &:not(:has(.indexTable-body-row-cell)) {
+            padding: 0;
+            margin-bottom: 0;
+        }
+        &:last-child {
+            margin-bottom: 0;
+        }
+    }
+
+    // checkbox, title and subtotal need to stay on the same row
+    .indexTable-body-row-transparentCell {
+        grid-row: 1;
+    }
+
+    // We only need one ::before for the card apparence and one ::after for the possible outline
+    .indexTable-body-row-cell {
+        position: static;
+        grid-column-start: 1;
```

Comment:
@BertrandPodevin : I’m wondering why each cell is forced to start at column 1.

Maybe I’m missing something, but should we make this a `var(--components-indexTable-cell-colum-start, auto)` to allow alignment on any column specified by `--components-indexTable-row-responsive-grid-template-*`, without using `!important`?